### PR TITLE
[SDPA-1331] Print Styles.

### PR DESCRIPTION
**Pull request by alan-cole**
**JIRA issue:** https://digital-engagement.atlassian.net/browse/SDPA-1331

### Changed

* Adding print styles to components.
* Updating print page to include all components.
* Rendered cards. 
* Updated tests.
* Set all print text to be black. Hide some components.
* Add page breaks on news listing.
* Prevent grid css on print.
* Underlines on links.
* Improve button layout (left aligned).
* Remove CardNavigationFeatured (match as per other hidden cards).
* Update snapshots / docs.

This ticket has been in development for a particularly long time with a lot of discussion and, unfortunately, many limitations. The implementation here is a compromise, to avoid overly large structural changes to the existing code, but still aiming for legibility in print. I expect the result of this testing will require some discussion and further revision, so feel free to reach out with questions and suggestions.

### Screenshots

[Safari-Print.pdf](https://github.com/dpc-sdp/ripple/files/3037169/Safari-Print.pdf)
[Firefox-Print.pdf](https://github.com/dpc-sdp/ripple/files/3037170/Firefox-Print.pdf)
[Chrome-Print.pdf](https://github.com/dpc-sdp/ripple/files/3037171/Chrome-Print.pdf)

### CODE REVIEW

File: packages/Atoms/Global/scss/settings/_print.scss [hunk 1]
```diff
@@ -0,0 +1,20 @@
+// Add visible URL after links.
+// a[href]:after {
+//   content: ' (' attr(href) ')';
+//   font-weight: normal;
+//   font-size: $base-font-size;
+// }
+
+// Only display useful links.
+// a[href^='javascript:']:after,
```
Comment (dylankelly) [inline packages/Atoms/Global/scss/settings/_print.scss]: Remove these comments?

File: packages/Molecules/Alert/AlertBase.vue [hunk 2]
```diff
@@ -67,6 +67,10 @@ $rpl-alert-base-icom-margin: 0 $rpl-space-4 0 0 !default;
   display: flex;
   align-items: center;
 
+  @media print {
```
Comment (dylankelly) [inline packages/Molecules/Alert/AlertBase.vue]: this could also be a useful mixin

File: packages/Atoms/Global/scss/settings/_print.scss (hunk 1 above)
Comment (alan-cole) [reply packages/Atoms/Global/scss/settings/_print.scss]: Fixed.

File: packages/Atoms/Global/scss/components/_button.scss [hunk 3]
```diff
@@ -48,13 +59,17 @@
...
+    @include rpl_text_color($rpl-button-active-text-color);
   }
 }
 
 @mixin rpl_button_secondary {
+  @include rpl_text_color($rpl-button-secondary-text-color);
   background-color: $rpl-button-secondary-background-color;
-  color: $rpl-button-secondary-text-color;
+
+  @media print {
```
[the remark this answers is not in the document]
Comment (alan-cole) [reply packages/Atoms/Global/scss/components/_button.scss]: Updated to be a mixin.

File: packages/Molecules/Alert/AlertBase.vue (hunk 2 above)
Comment (alan-cole) [reply packages/Molecules/Alert/AlertBase.vue]: Updated to be a mixin.

**Pull request merged**